### PR TITLE
Add session deletion to release flow

### PR DESCRIPTION
**Pull request by RyanMacG**
This allows us to build and deploy the session deletion payload manually

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,10 +10,12 @@ test:
 
 compile:
 	pip3 install --no-cache-dir -r requirements.txt -t vendor
-	cd vendor; zip -r ../deletion-payload.zip ../user_deletion.py *; cd -
+	cd vendor; zip -r ../user_deletion.zip ../user_deletion.py *; cd -
+	cd vendor; zip -r ../session_deletion.zip ../session_deletion.py *; cd -
 
 deploy:
-	aws lambda update-function-code --zip-file fileb://deletion-payload.zip --function-name user_deletion --region eu-west-2
+	aws lambda update-function-code --zip-file fileb://user_deletion.zip --function-name user_deletion --region eu-west-2
```
Comment (adrianclay) [inline Makefile]: Do you think it's worth splitting into 2 separate recipes? `compile-user-deletion` & `compile-session-deletion` etc?

**Comment by adrianclay** [comment]
Completely unrelated, but I've just re-read the README, it's a bit vague about what data we are deleting with this repository.  Would be useful to give a brief explanation there.

File: Makefile (hunk 1 above)
Comment (RyanMacG) [reply Makefile]: Currently, I can't see these being run separately so I think we should make that change if and when we need it.

**Comment by RyanMacG** [comment]
I've updated the README in another PR

**Pull request merged**